openfl-workspace/torch_cnn_mnist/README.md

```diff
@@ -138,7 +174,7 @@ fx plan initialize --input_shape [1,28,28] --aggregator_address localhost
 We can now perform a test run with the following commands for creating a local PKI setup and starting the aggregator and the collaborators on the same machine:
 
 ```
-cd ~/openfl/openfl-tutorials/taskrunner/
+cd ~/openfl/openfl-workspace/torch_cnn_mnist/
```

Comment:
same here - `~/openfl-quickstart/fl_workspace`

Reply:
fixed in c9bb19088383b978a147804539dee0d7dd37b515